server/controllers/index.js

```diff
@@ -27,16 +28,46 @@ export const article = async(ctx, next) => {
 
 export const articles = async(ctx, next) => {
   const wpPosts = await getPosts(32);
-  const promos = postsToPromos(wpPosts.data);
+  const items = postsToPromos(wpPosts.data);
+  const {total} = wpPosts;
+  const series: Series = {
+    url: '/articles',
+    name: 'Articles',
+    total,
+    items
+  };
+  ctx.render('pages/list', {
+    pageConfig: createPageConfig({
+      title: 'Articles',
+      inSection: 'explore'
+    }),
+    list: series
+  });
 
-  // TODO: We might change this to `index`
-  ctx.render('pages/articles', {
+  return next();
+};
+
+export const series = async(ctx, next) => {
+  const {id} = ctx.params;
+  const wpPosts = await getPosts(32, {category: id});
+  const items = postsToPromos(wpPosts.data);
+  // TODO: So So nasty
+  const {name, description} = wpPosts.data.first().series[0];
+  const {total} = wpPosts;
+  const series: Series = {
```

Comment:
https://github.com/wellcometrust/wellcomecollection.org/pull/546/files/aa236eb7768bc053b9d2c3ec4db5ca3ec75ba492#diff-6f46a566e77b008a339b5a3dfecbdc96L1

All flow is stripped - but you need to add it to do the checking... there was a separate PR for this post launch.